### PR TITLE
Added ascension materials for some characters

### PR DESCRIPTION
**Pull request by myun11**
Added ascension materials for Albedo, Alhaitham, Aloy, Amber, Arataki Itto, Ayaka, Ayato, Baizhu, Barbara, Beidou, Bennett.

The json structure is separated by ascension level, the name of the material and the quantity of the material. I can do this for all characters later on.

### CODE REVIEW

File: assets/data/characters/dori/en.json [hunk 1]
```diff
@@ -167,5 +167,135 @@
     }
   ],
   "vision_key": "ELECTRO",
-  "weapon_type": "CLAYMORE"
+  "weapon_type": "CLAYMORE",
+  "ascension_materials": {
+    "level_20": [
+      {
+        "name": "Shivada Jade Sliver",
```
Comment (man90es) [inline assets/data/characters/dori/en.json]: Dori uses Vajrada Amethyst

File: assets/data/characters/dori/en.json [hunk 2]
```diff
@@ -167,5 +167,135 @@
...
+        "value": 1
+      },
+      {
+        "name": "Calla Lily",
```
Comment (man90es) [inline assets/data/characters/dori/en.json]: Dori uses Kalpalata Lotus

File: assets/data/characters/dori/en.json [hunk 3]
```diff
@@ -167,5 +167,135 @@
...
+        "value": 3
+      },
+      {
+        "name": "Firm Arrowhead",
```
Comment (man90es) [inline assets/data/characters/dori/en.json]: Dori uses Red Satin

**Review by man90es — CHANGES_REQUESTED** [review]

**Comment by myun11** [comment]
Fixed Dori's ascension_materials.

**Review by man90es — APPROVED** [review]
Looks good to me, waiting for @M4GICB's review

**Review by M4GICB — APPROVED** [review]
Data all looks good. I only have 1 suggestion:
- `phase_1` - `phase_6` would make more sense than `level_20` - `level_80`
- This would make it so that the data shows the cost of unlocking `ascension phase x` rather than the cost of surpassing `level x`

If others disagree i am fine with this being merged though.

**Comment by man90es** [comment]
> * `phase_1` - `phase_6` would make more sense than `level_20` - `level_80`
> * This would make it so that the data shows the cost of unlocking `ascension phase x` rather than the cost of surpassing `level x`

Idk honestly, I think both ways is fine

**Comment by myun11** [comment]
I think it's fine. Can always reflect that on the front end.

**Pull request merged**